client/a8c-for-agencies/components/form/field/index.tsx

```diff
@@ -9,17 +9,24 @@ type Props = {
 	description?: string;
 	isOptional?: boolean;
 	children: ReactNode;
+	isRequired?: boolean;
 };
 
-export default function FormField( { label, sub, children, description, isOptional }: Props ) {
+export default function FormField( {
+	label,
+	sub,
+	children,
+	description,
+	isOptional,
```

Comment:
`isOptional` Maybe we could rename this by `showOptionalLabel`

Reply:
This one is my fault. 🙈